<ui version="4.0" >
 <class>MMSReceivedDialog</class>
 <widget class="QDialog" name="MMSReceivedDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>MMS Received</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneNumberLabel" >
     <property name="text" >
      <string>Phone number:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mmsPhoneNumberEdit" />
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Subject:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mmsSubjectEdit" />
   </item>
   <item>
    <widget class="QLabel" name="messageLabel" >
     <property name="text" >
      <string>Message:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="mmsBodyEdit" />
   </item>
   <item>
    <widget class="QLabel" name="attachmentPath" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MMSReceivedDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
